src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supply = 0;
+    private int buy = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String [] dataFromFile = value.split(",");
+                if (dataFromFile[0].equals("supply")) {
+                    supply = supply + Integer.parseInt(dataFromFile[1]);
+                }
+                if (dataFromFile[0].equals("buy")) {
+                    buy = buy + Integer.parseInt(dataFromFile[1]);
+                }
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        writeToFile(supply, buy, toFileName);
```

Comment:
public methods should mostly contain easy-to-understand logic, so you extract complicated logic into private methods.
In this task, we want you to have readFromFile(), createReport(), and writeToFile() methods inside getStastistic.